data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg

```diff
@@ -325,6 +325,74 @@ A full list of abilities and weapons specials, along with traits, may be found i
+                [not]
+                    id=$unit.id
+                [/not]
+                {EVERYWHERE}
+            [/filter]
+            variable=other_healer
+        [/store_unit]
+
+        [if]
+            [variable]
+                name=other_healer.type
+                equals=Elvish Shaman
+            [/variable]
+            [then]
+                [message]
+                    speaker=$unit.id
+                    # po: both speaker and $other_healer are elvish shamans.
+                    # If there are allied sides then this isn't accurate, but at this stage of the tutorial allied sides haven't been introduced yet.
+                    message= _ "Both Shaman $other_healer.name and I can heal other units. However, each unit can only receive healing from one healer — even if we’re both next to a unit then that unit will still only heal 4 hitpoints per turn."
+                [/message]
+                [message]
+                    speaker=$unit.id
+                    # This line could be said by the other healer, but there's only one shaman portrait available, and it would be confusing to have dialog between two units with identical portraits.
+                    message= _ "If either of us advances to become a Druid, then a unit next to both of us will receive the Druid’s advanced healing. A unit in a village will receive healing from the village, neither a Shaman nor a Druid can improve on that."
+                [/message]
+                [message]
+                    speaker=Galdrad
+                    message= _ "Having more than one healer will mean that you can heal more units, and in separate areas of the battlefield. They can also heal each other. The powerful healing ability of their Elvish Druid advancement can be very helpful in campaigns, but they’re unlikely to gain that much experience during the tutorial — for now, concentrate on keeping your shamans alive so they can continue to heal your other troops."
+                [/message]
+            [/then]
+            [else]
+                [message]
+                    speaker=$unit.id
+                    # po: the speaker is a shaman, but $other_healer is a druid or (very unlikely) a Shyde.
+                    # If there are allied sides then this isn't accurate, but at this stage of the tutorial allied sides haven't been introduced yet.
+                    message= _ "Druid $other_healer.name and I can heal other units. However, each unit can only receive healing from one healer — if we’re both next to a unit then I can’t add any more on top of Druid $other_healer.name|'s advanced healing skills."
+                [/message]
+                [message]
+                    speaker=$other_healer.id
+                    message= _ "When $unit.name becomes a Druid, a unit next to both of us will still only receive one Druid’s healing. A unit in a village will receive healing from the village, neither a Shaman nor a Druid can improve on that."
```

Comment:
Again, the first sentence here duplicates the information in the previous line. I think we could merge them, and say just once: _a unit is healed by only one healer per turn. If a unit is adjacent to multiple healers, the most powerful one of them heals the unit._ 

(Also, there's a comma splice, but I'll put off discussing that level of detail until the flow of the dialog is settled.)

Reply:
I'm in favour of splitting it with the first message being (current situation) and the second one being (after the next level-up).

I'm moving to having both lines said by the recruited unit, even in the case that $other_healer is already a druid with a different portrait. The idea was that the higher-level one would give a cheery "you'll gain experience" message to the new recruit, but in my attempts to rewrite it that doesn't feel useful any more.

Reply:
It might be hard to work such a cherry message in there, given that we have a self-imposed quota of at most three lines of dialog, and four or five different things to teach :)